server/http_integration_test.go

```diff
@@ -2452,6 +2612,29 @@ func getAccessControlMessages(hook *logrustest.Hook) string {
 	return ""
 }
 
+func getAllDaemonMessages(hook *logrustest.Hook) []string {
+	var msgs []string
+	for _, entry := range hook.AllEntries() {
+		if entry.Message != "" {
+			msgs = append(msgs, entry.Message)
+		}
+	}
+
+	return msgs
+}
+
+func getAccessLogUrl(hook *logrustest.Hook) string {
```

Comment:
missing filter for log type?

AllEntries could be type access, backend, daemon.